header.hpp.in

```diff
@@ -0,0 +1,8 @@
+/* ************************************************************************
+ * Copyright 2019-2022 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+/* The Wrapper template Configuration & settings
+ */
+#warning "This file is deprecated and will be removed in future releases. Use the include path /opt/rocm-ver/include and prefix with rocprim."
```

Comment:
Sorry missed this should it be ..."prefix all includes with rocprim/"

Reply:
Hmmm but on windows that is wrong, could just use include path /opt/rocm-ver/include/rocprim as well, right?